### DISCUSSION TITLE
Last Update

### DISCUSSION THREAD

**Discussion by dusankonder** [wills106/homeassistant-solax-modbus#657] [General]
Hi, after last update plugin_sofar.py i have some sensors unavailable.
 PV Current 1
 PV Current 2
PV Power 1
PV Power 2
PV Voltage 1
PV Voltage 1
When i roll back from this entity's last change, then works again with allowedtypes = HYBRID | PV | GEN,

**Comment by wills106** [comment]
I'll correct it in the next release. I configured MPPT 1 & 2 wrong when adding in the extra MPPT's for larger Inverters.

**Reply by wills106** [reply]
Should be corrected in [2023.12.4](https://github.com/wills106/homeassistant-solax-modbus/releases/tag/2023.12.4)

**Reply by dusankonder** [reply]
Thanks. everythink is OK. :)
